### PR TITLE
refactor(php8): remove deprecation warnings

### PR DESCRIPTION
**Pull request by rowhel95**
Forgot these

### CODE REVIEW

File: src/Support/Collection.php [hunk 1]
```diff
@@ -1778,7 +1778,7 @@ public function toJson($options = 0)
      *
      * @return \ArrayIterator
      */
-    public function getIterator()
+    public function getIterator(): Traversable
```
Comment (EdieLemoine) [inline src/Support/Collection.php:1781]: Why not `ArrayIterator`?

**Review by EdieLemoine — CHANGES_REQUESTED** [review]

File: src/Support/Collection.php (hunk 1 above)
Comment (rowhel95) [reply src/Support/Collection.php:1781]: Because I'm not sure if the return type needs an exact match or not

Comment (EdieLemoine) [reply src/Support/Collection.php:1781]: If you change it to `ArrayIterator` I'll approve the PR

**Comment by it-can** [comment]
Any news on this?

**Pull request merged**